scripts/gofumpt.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Format staged Go files with `gofumpt`
+
+# Redirect output to stderr.
+exec 1>&2
```

Comment:
Why not let the operator redirect streams the way they see fit? Especially as the script outputs to `>&2` as if it makes a difference after this.

Reply:
Fixed in https://github.com/cometbft/cometbft/pull/2049/commits/0a73d731aaa9dfba29687dc28eaca1f64f95489b